manifests/p/ppy/osu/2024.412.1/ppy.osu.locale.en-US.yaml

```diff
@@ -0,0 +1,37 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: ppy.osu
+PackageVersion: 2024.412.1
+PackageLocale: en-US
+Publisher: ppy Pty Ltd
+PublisherUrl: https://github.com/ppy
+PublisherSupportUrl: https://github.com/ppy/osu/issues
+PackageName: osu!
+PackageUrl: https://github.com/ppy/osu
+License: MIT
+LicenseUrl: https://github.com/ppy/osu/raw/master/LICENCE
+Copyright: Copyright (c) 2024 ppy Pty Ltd
+CopyrightUrl: https://github.com/ppy/osu/raw/master/LICENCE
+ShortDescription: rhythm is just a *click* away!
+Description: |-
+  A free-to-win rhythm game. Rhythm is just a click away!
+
+  This is the future – and final – iteration of the osu! game client which marks the beginning of an open era! Currently
+  known by and released under the release codename "lazer". As in sharper than cutting-edge.
+Tags:
+- beatmap-parser
+- game
+- hacktoberfest
```

Comment:
Not sure this one should be here, doesn't really make sense for an end user.

Reply:
These tags are automatically inspected by _Komac_ from the repository homepage.

![image](https://github.com/microsoft/winget-pkgs/assets/49941141/39136d78-89da-4e5a-bcca-48354a577f0f)

I would preffer attaching them here. Users can search this package by tags via `winget search`. More accurate informations in the manifest, more users can find it.

Reply:
I mean specifically `hacktoberfest`.

I've updated the repository tags, so please re-fetch them.